python/ray/log_monitor.py

```diff
@@ -89,18 +90,20 @@ def close_all_files(self):
 
     def update_log_filenames(self):
         """Update the list of log files to monitor."""
-        log_filenames = os.listdir(self.logs_dir)
-
-        for log_filename in log_filenames:
-            full_path = os.path.join(self.logs_dir, log_filename)
-            if full_path not in self.log_filenames:
-                self.log_filenames.add(full_path)
+        # we only monior worker log files
+        log_file_paths = glob.glob("{}/worker*[.out|.err]".format(
```

Comment:
isn't there other log files that the log_monitor tracks? like the monitor log, the raylet log?